tools/evmc/main.cpp

```diff
@@ -0,0 +1,89 @@
+// EVMC: Ethereum Client-VM Connector API.
+// Copyright 2019 The EVMC Authors.
+// Licensed under the Apache License, Version 2.0.
+
+#include <CLI/CLI.hpp>
+#include <evmc/loader.h>
+#include <evmc/mocked_host.hpp>
+
+#include "utils.hpp"
+
+int main(int argc, const char** argv)
+{
+    using namespace evmc;
+
+    CLI::App app{"EVMC tool"};
+    const auto& version_flag = *app.add_flag("--version", "Print version information");
+
+    std::string vm_config;
+    std::string code_hex;
+    evmc_message msg{};
+    msg.gas = 1000000;
+    auto rev = EVMC_ISTANBUL;
+
+    auto& run_cmd = *app.add_subcommand("run", "Execute EVM bytecode");
+    run_cmd.add_option("code", code_hex, "Hex-encoded bytecode")->required();
+    run_cmd.add_option("--vm", vm_config, "EVMC VM module")->required()->envname("EVMC_VM");
```

Comment:
Does it take the value from environment variable by default? Also maybe say this in the help message if it doesn't do it automatically.

Reply:
It uses the env var if `--vm` not provided.